packages/react-ui-core/src/Form/RadioGroup.js

```diff
@@ -108,29 +149,68 @@ export default class RadioGroup extends Component {
       className,
       theme,
       name,
+      label,
+      error,
+      hint,
+      disabled,
+      required,
       fields,
       hideInputElement,
       onChange,
       allowUnselect,
       onUnselect,
       variant,
-      ...props
+      children,
+      ...rest
     } = this.props
 
+    const id = rest.id || this.uniqueId
+    const invalid = rest.invalid || !!error
+
+    const props = {
+      label: null,
+      error: null,
+      hint: null,
+    }
+
     if (!this.fields.length) return null
 
+    if (label) {
+      props.label = createElement(...parseArgs(label, Label, {
```

Comment:
is there an advantage of using createElement as opposed to straight jsx?

Reply:
No, not really. It's the same, under the hood jsx converts to this anwyay.